todo/src/reducer/reducers.jsx

```diff
@@ -0,0 +1,53 @@
+import { ADD_TASK, TOGGLE_COMPLETED, DELETE_COMPLETED, DELETE_ITEM } from '../action/actions.jsx';
+
+const initialState = {
+	todo: [{
+		name: 'test name',
+		id: Date.now(),
+		completed: false,
+	}, {
+		name: 'secondary test',
+		id: Date.now() + 4397,
+		completed: false,
+	},]
+}
+
+export default (state = initialState, action) => {
+	switch (action.type){
+		case ADD_TASK:
+		return {
+			...state,
+				todo: [
+					...state, {
+						name: action.payload,
+						id: Date.now(),
+						completed: false,
```

Comment:
This won't work because you can't spread `state` (an object) inside an array. You likely meant `...state.todo`.